docs/content/en/docs/implementing/slo/_index.md

```diff
@@ -0,0 +1,108 @@
+---
+title: Analisis
+description: Understand Analysis in Keptn and how to use them
+weight: 150
```

Comment:
Is this where it logically belongs in the "User Guides" section -- after "Restart an application" and before "Evaluations"?  This seems odd...

Reply:
I am open to suggestions, I just copied it from an other file 😸

Reply:
Okay, cool.  I'll deal with this after this PR is merged.

Reply:
Why not put it into the correct place right away?

Reply:
yeah so where should we put it?

Reply:
i suggest to put it after “working with keptn tasks”
and i would change the title from “Analysis” to “Define SLOs/SLIs with Analyses” or somehting like that. more a usecase title that just what it is